cryptarithm.py

```diff
@@ -138,7 +138,7 @@ def build_dqm(variable_list: List[LetterVariable], coefficient_map: dict) -> DQM
     response = LeapHybridDQMSampler(
         solver="hybrid_discrete_quadratic_model_version1", auto_scale=False
     ).sample_dqm(
-        dqm, time_limit=5, compress=True
+        dqm, time_limit=5, compress=True, label="Example - Cryptarithmetic"
```

Comment:
I also believe the `compress` argument won't apply here as per the documentation for `sample_dqm` and the number of biases in these problems.